pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -905,6 +906,7 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, useEmulat
 		IsBlockPVC:     isBlockPVCMap,
 		DiskType:       diskInfo,
 		SRIOVDevices:   getSRIOVPCIAddresses(vmi.Spec.Domain.Devices.Interfaces),
+		SMBios:         options.VirtualMachineSMBios,
```

Comment:
At least in theory options can be nil here, please catch that. It can happen when the request is coming from an older virt-handler. That _should_ not happen, because handler _should_ be updated before launcher, but "should" doesn't mean it will never happen.

Reply:
I agree with the idea, but we already check that in the converter here https://github.com/kubevirt/kubevirt/pull/2472/files#diff-caee6d384498fde52fde15891ac547aaR691
@slintes Do you think we need additional check here?

Reply:
converter is unrelated. I mean that `options` can be nil here, so options.VirtualMachineSMBios will fail during runtime

Reply:
I agree it's probably better to verify. At the moment there is no scenario where we don't populate the options, but it may change in the future.

Reply:
Added